contributors/devel/sig-instrumentation/instrumentation.md

```diff
@@ -90,7 +90,8 @@ apply additionally.
 
 ## Naming 
 
-Metrics added directly by application or package code should have a unique name. 
+General [metric and label naming apply](https://prometheus.io/docs/practices/naming/),
```

Comment:
`metric and label naming best practices apply`?

Reply:
oops yes